test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -345,7 +346,14 @@ Describe "Import-Module from CompatiblePSEditions-checked paths" -Tag "CI" {
 
             # after WinCompat cleanup local $PWD changes should not cause errors
             Remove-module $ModuleName -Force
-            Pop-Location
+            try
```

Comment:
Actually, I think all code after the first 'Push-Location' should be in the try.  If any exception is thrown during the test the original location will be reverted through the finally 'Set-Location'.

Reply:
Updated.